bcgov/Unity
# Dev To Main Release 2024-05-01

QA Testable:
- [AB#23712](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23712) Supplier & Site maintenance & info in the applicant info tab
- [AB#9230](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/9230) Ability to link submission to one another
- [AB#24066](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24066) fix status action update error (Already hot fixed in PROD)
- [AB#23402](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23402) Account coding info available and editable under Payment Configuration
- [AB#24004](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24004) UX2.0 Theme (initial changes, more to come)
- [AB#23980](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23980) Contract number allow alphanumeric chars
- [AB#23803](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23803) Update Application# column title (Submission #)
- [AB#23940](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23940) Basic input filter fields for Dashboards
- [AB#24243](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24243) (Bugfix) Sync the edit behavior Request Amount and Total Project budget on both tabs (Review & Assess / Project Info)
- [AB#23448](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23448) Add place field to Project Info 
- [AB#23983](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23983) Update labels for additional contacts

QA - To be verified as still WIP:
- [AB#23715](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23715) Add site dropdown to payment requests creation
- [AB#23721](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23721) Batch payments list
- [AB#23008](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23008) Payments list

Non QA Testable:
- [AB#13332](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/13332) .NET Upgrade (To V8 and ABP V8.1)
- [AB#24121](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24121) Cookie updates on no grant program error page
- [AB#12630](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/12630) Change default serilog storage settings
- [AB#24010](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24010) Update project readme contents
- [AB#23646](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/23646) Add health check probes
- [AB#24295](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24295) Add the Flex (Dynamic scoring and fields) module (Still WIP)
- [AB#24172](https://tfs.econ.gov.bc.ca/ECON/12d03e79-76dc-4213-a504-6fdfa588b4f6/_workitems/edit/24172) Application offline page
